### PR TITLE
TeamA F22: External state estimation

### PR DESCRIPTION
**Pull request by John-HarringtonNZ**
This is an update of the robotics-knowledge-base website, with content relating to GPS-lacking sensing systems including UltraWideband beacons and robotic total stations. This acts as Team A's Fall 2022 contribution as part of the MRSD Project Course (16-682).

### CODE REVIEW

File: wiki/sensing/robotic-total-stations.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+# You should set the date the article was last updated like this:
+date: 2022-12-05 # YYYY-MM-DD
+# This will be displayed at the bottom of the article
+# You should set the article's title:
+title: Robotic Total Station (Leica TS16)
+# The 'title' is automatically displayed at the top of the page
+# and used in other parts of the site.
+---
+
+## Introduction
```
Comment (smbryan) [inline wiki/sensing/robotic-total-stations.md:13]: The first paragraph (or the first 100 words) becomes the page description. The blurb for this page will just read "## Introduction"...

I'd recommend dropping this line and keeping the first page as-is. 

CC: @nevalsar

File: wiki/sensing/ultrawideband-beacon-positioning.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+# This will be displayed at the bottom of the article
+# You should set the article's title:
+title: DWM1001 UltraWideband Positioning System
+# The 'title' is automatically displayed at the top of the page
+# and used in other parts of the site.
+---
+
+## Introduction
+
+This page will go into detail about to get started with UltraWideband positioning, particuarly with the [DWM1001-Dev module](https://www.qorvo.com/products/p/DWM1001-DEV).
```
Comment (smbryan) [inline wiki/sensing/ultrawideband-beacon-positioning.md:15]: Same issue here. The link would also be displayed in its native markdown if left. Please rewrite to omit any headers (e.g. `## Introduction`) or Markdown. 

Your "Externally Referenced" article below is a particularly strong intro. I'd model follow that example.

CC: @nevalsar

**Review by smbryan — CHANGES_REQUESTED** [review]
I appreciate the fresh content here. Some minor housekeeping before we can merge this in.

**Review by nevalsar — CHANGES_REQUESTED** [review]
(See earlier review comments)

**Comment by nevalsar** [comment]
Contributor unresponsive - will raise these changes in a separate MR fixing issues.

**Pull request closed without merging**